### PR TITLE
Add an important remark for user

### PR DESCRIPTION
**Pull request by michou214**
**Description**
Update the Robotino 3 documentation

**Documentation**
[Modification visible here](https://cyberbotics.com/doc/guide/robotino3?version=michou214-patch-1)

### CODE REVIEW

File: docs/guide/robotino3.md [hunk 1]
```diff
@@ -49,6 +49,23 @@ Robotino3 {
 
 You will find the following sample in this folder: "[WEBOTS\_HOME/projects/robots/festo/robotino3/worlds](https://github.com/cyberbotics/webots/tree/master/projects/robots/festo/robotino3/worlds)".
 
+> **Note:** To have the right behavior of the robot in your simulation, you have to add this portion of code in your world file in the `WorldInfo` field.
```
Comment (DavidMansolino) [inline docs/guide/robotino3.md]:
```suggestion
> **Note:** For the mecanum wheels to behave correctly, the following [ContactProperties](../reference/contactproperties.md) should be added in the `WorldInfo` field.
```

**Comment by michou214** [comment]
:+1:

File: docs/guide/robotino3.md [hunk 2]
```diff
@@ -49,6 +49,23 @@ Robotino3 {
 
 You will find the following sample in this folder: "[WEBOTS\_HOME/projects/robots/festo/robotino3/worlds](https://github.com/cyberbotics/webots/tree/master/projects/robots/festo/robotino3/worlds)".
 
+> **Note:** For the mecanum wheels to behave correctly, the following [ContactProperties](../reference/contactproperties.md) should be added in the `WorldInfo` field.
```
Comment (omichel) [inline docs/guide/robotino3.md]:
```suggestion
> **Note:** For the mecanum wheels to behave correctly, the following [ContactProperties](../reference/contactproperties.md) should be added in the `contactProperties` field of the `WorldInfo` node:
```

Comment (DavidMansolino) [reply docs/guide/robotino3.md]: I would even add `WorldInfo` as a link ;-)

File: docs/guide/robotino3.md [hunk 3]
```diff
@@ -49,6 +49,23 @@ Robotino3 {
 
 You will find the following sample in this folder: "[WEBOTS\_HOME/projects/robots/festo/robotino3/worlds](https://github.com/cyberbotics/webots/tree/master/projects/robots/festo/robotino3/worlds)".
 
+> **Note:** For the mecanum wheels to behave correctly, the following [ContactProperties](../reference/contactproperties.md) should be added in the `contactProperties` field of the `WorldInfo` node:
```
Comment (DavidMansolino) [inline docs/guide/robotino3.md]:
```suggestion
> **Note:** For the mecanum wheels to behave correctly, the following [ContactProperties](../reference/contactproperties.md) should be added in the `contactProperties` field of the [WorldInfo](../reference/worldinfo.md) node:
```

**Pull request merged**